src/components/Calculator.js

```diff
@@ -1,11 +1,15 @@
 import CalculatorButtons from './CalculatorButtons';
+import Quote from './Quote';
 
 function Calculator() {
   return (
-    <div className="calculator">
-      <form action="">
-        <CalculatorButtons />
-      </form>
+    <div>
+      <Quote />
```

Comment:
Rather than importing it in the Calculator component its a project requirement The `<Quotes>` component should be imported into the main component like follow:-

```
import Calculator from "./components/Calculator"
import Quotes from "./components/Quotes"

function App() {
  return (
    <>
      {/* Rest of JSX */}
      <Calculator />
      <Quotes />
    </>
  )
}

export default App
```